charts/spire/charts/spire-server/values.yaml

```diff
@@ -445,38 +445,68 @@ controllerManager:
     - local-path-storage
 
   identities:
-    ## @param controllerManager.identities.enabled Flag to enable default identities for controller manager
-    enabled: true
-
-    ## @param controllerManager.identities.spiffeIDTemplate Spiffe ID template for identities
-    spiffeIDTemplate: spiffe://{{ .TrustDomain }}/ns/{{ .PodMeta.Namespace }}/sa/{{ .PodSpec.ServiceAccountName }}
-    ## @param controllerManager.identities.podSelector [object] Selector for pods to issue identity
-    podSelector: {}
-      # matchLabels:
-      #   spiffe.io/spiffe-id: "true"
-    ## @param controllerManager.identities.namespaceSelector [object] Selector for namespacs to issue identity
-    namespaceSelector: {}
-      # matchLabels:
-      #   spiffe.io/spiffe-id: "true"
-    ## @param controllerManager.identities.dnsNameTemplates [array] DNS name template for issued identities
-    dnsNameTemplates: []
-      # - '{{ index .PodMeta.Labels "app.kubernetes.io/name" }}.{{ .PodMeta.Namespace }}.svc.cluster.local'
-    ## @param controllerManager.identities.federatesWith [array] Other Spire server URLs for identity federation
-    federatesWith: []
-    # - example.io
-    # - example.ai
-    ## @param controllerManager.identities.workloadSelectorTemplates [array] Templates to produce selectors that apply to a given workload before it will receive an ID
-    workloadSelectorTemplates: []
-    ## @param controllerManager.identities.ttl Indicates an upper-bound time-to-live for X509 SVIDs. If unset, the cluster default will be chosen.
-    ttl: ""
-    ## @param controllerManager.identities.jwtTTL Indicates an upper-bound time-to-live for JWT SVIDs. If unset, the cluster default will be chosen.
-    jwtTTL: ""
-    ## @param controllerManager.identities.admin Indicates any pod matched by this identity will be an admin. Use this with extreme care.
-    admin: false
-    ## @param controllerManager.identities.downstream Set if this spire instance is a root server and the workloads are downstream servers.
-    downstream: false
-    ## @param controllerManager.identities.autoPopulateDNSNames Auto populate DNS names from services attached to pods
-    autoPopulateDNSNames: false
+    clusterSPIFFEIDs:
+      # NOTE you can add multiple uniquely named entries to create multiple ClusterSPIFFEID objects. See example below.
```

Comment:
What do you think of moving this above clusterSPIFFEIDs and putting the @param on it so it shoes up in rendered docs?

Reply:
Can try it... not sure what it will do with all the stuff inside it. lets see...

Reply:
I tried setting:
>     ## @param controllerManager.identities.clusterSPIFFEIDs Specify one or more ClusterSPIFFEID objects
and it fails with:

```
ERROR: Metadata provided for non existing key: controllerManager.identities.clusterSPIFFEIDs
/home/kfox/.npm-packages/lib/node_modules/@bitnami/readme-generator-for-helm/lib/checker.js:93
    throw new Error('ERROR: Wrong metadata!');
```

Reply:
If I add the type:
```
## @param controllerManager.identities.clusterSPIFFEIDs [object] Specify one or more ClusterSPIFFEID objects
```

it fixes that part, then it breaks later
```
ERROR: Metadata provided for non existing key: controllerManager.identities.clusterSPIFFEIDs.default.enabled
ERROR: Metadata provided for non existing key: controllerManager.identities.clusterSPIFFEIDs.default.spiffeIDTemplate
ERROR: Metadata provided for non existing key: controllerManager.identities.clusterSPIFFEIDs.default.ttl
ERROR: Metadata provided for non existing key: controllerManager.identities.clusterSPIFFEIDs.default.jwtTTL
ERROR: Metadata provided for non existing key: controllerManager.identities.clusterSPIFFEIDs.default.admin
ERROR: Metadata provided for non existing key: controllerManager.identities.clusterSPIFFEIDs.default.downstream
ERROR: Metadata provided for non existing key: controllerManager.identities.clusterSPIFFEIDs.default.autoPopulateDNSNames
```

Reply:
Yeah, I don't think the doc tool supports commenting both at a branch, and the leaves.

Reply:
Oh well, thanks for trying.